Makefile

```diff
@@ -0,0 +1,59 @@
+RELEASE ?= `grep -1 -A 0 -B 0 '<version>' pom.xml | head -n 1 | awk '{print $1}' | sed -e 's/.*<version>//' | sed -e 's/<\/version>.*//'`
+DOCKER_REPO ?= mesos
+MIRROR ?=
+STORM_URL ?=
+JAVA_PRODUCT_VERSION ?= 7
+
+all: help
+
+help:
+	@echo 'Options available:'
+	@echo '  make images STORM_RELEASE=0.10.0 MESOS_RELEASE=0.28.0 DOCKER_REPO=mesos' 
+	@echo '  make push   STORM_RELEASE=0.10.0 MESOS_RELEASE=0.28.0 DOCKER_REPO=mesos'
+	@echo ''
+	@echo 'ENV'
+	@echo '  STORM_RELEASE 				The targeted release version of Storm'
+	@echo '                 				Default: 0.10.0'
+	@echo '  MESOS_RELEASE      		The targeted release version of MESOS'
```

Comment:
the formatting is off here -- I'm guessing you're using tabs and have a tab setting which makes this line up nicely for *your* view here?

(I realize tabs are annoyingly required in Makefiles, but I suggest limiting their use to the required locations, and not proliferating the tab contagion.  Don't get me started on go-lang's love of tabs and autoformatting! 😛 )